website/www/site/content/en/get-started/quickstart-java.md

```diff
@@ -297,7 +297,7 @@ TODO: document Spark on Gradle: https://github.com/apache/beam/issues/21502
 {{< runner dataflow >}}
 gradle clean execute -DmainClass=org.apache.beam.examples.WordCount \
     --args="--project=<your-gcp-project> --inputFile=gs://apache-beam-samples/shakespeare/* \
-    --output=gs://<your-gcs-bucket>/counts" -Pdataflow-runner
+    --output=gs://<your-gcs-bucket>/counts --runner=DataflowRunner" -Pdataflow-runner
```

Comment:
do we still need `-Pdataflow-runner` here?

Reply:
yes it is required, as otherwise it gives error "Exception in thread "main" java.lang.IllegalArgumentException: Unknown 'runner' specified 'DataflowRunner', supported pipeline runners [DirectRunner, PortableRunner, TestUniversalRunner]"

Reply:
cc @Abacn

Reply:
yes. -Pdataflow-runner is a flag for maven pprofile. It adds beam-runners-google-cloud-dataflow-java to classpath; --runner=DayaflowRinner is a pipeline option.